Expensify/App
# Update inline code replace regex to not include code block

### Details
Update inline code replace regex to not include code block

### Fixed Issues
$ https://github.com/Expensify/App/issues/26966
PROPOSAL: https://github.com/Expensify/App/issues/26966#issuecomment-1745686470

### Tests
1. Open the app
2. Open any report
3. Send text in following format : `(```Test```)`
4. Observe that it displays correct value immediately after sending the message
5. Again edit the message and change the format back to: `(```Test```)`
6. Save and observe that it displays correct value immediately


### Offline tests
Same as above

### QA Steps
Same as above


### PR Author Checklist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)

### Screenshots/Videos
<details>
<summary>Android: Native</summary>
</details>

<details>
<summary>Android: mWeb Chrome</summary>
</details>

<details>
<summary>iOS: Native</summary>
</details>

<details>
<summary>iOS: mWeb Safari</summary>
</details>

<details>
<summary>MacOS: Chrome / Safari</summary>
</details>

<details>
<summary>MacOS: Desktop</summary>
</details>
